distributionviewer/views.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os.path
+
+from django.conf import settings
+from django.http import HttpResponse, HttpResponseServerError, JsonResponse
+
+from .api.models import Metric
+
+
+def ops_lbheartbeat(request):
+    return HttpResponse('ok')
+
+
+def ops_heartbeat(request):
+    try:
+        # Check the database by counting Metrics.
+        Metric.objects.count()
+        # TODO: Add checks for other dependencies as we add them
+        return HttpResponse('ok')
+    except:
+        return HttpResponseServerError('database failure')
+
+
+def ops_version(request):
+    # TODO: Generate this via TravisCI.
```

Comment:
I'll fix this in the next iteration when I actually do the TODO. It's a minor comment.